Cargo.toml

```diff
@@ -363,10 +363,6 @@ leveldb-sys = { git = "https://github.com/vectordotdev/leveldb-sys.git", branch
 chrono = { git = "https://github.com/vectordotdev/chrono.git", branch = "no-default-time-v0.4.22-1" }
 # Adds `Status::is_reset` test, remove after the next version is released.
 h2 = { git = "https://github.com/hyperium/h2.git", rev = "f6aa3be6719270cd7b4094ee1940751b5f4ec88e" }
-# Enables OpenSSL ENGINE support during cross compilation to musl
-# TODO remove this branch and instead use the openssl-src feature flag once it is available
-#    (see https://github.com/vectordotdev/vector/issues/13695)
-openssl-src = { git = "https://github.com/vectordotdev/openssl-src-rs.git", branch  = "enable_engine" }
```

Comment:
I don't think the release contains the changes we need ...

I think we need this change:

https://github.com/alexcrichton/openssl-src-rs/pull/151/files

Reply:
And then to add `force-engine` feature flag to our `Cargo.toml`

Reply:
Ah, doh, right, I do have to add the feature flag. I think this release _does_ include the changes though. It corresponds to https://github.com/alexcrichton/openssl-src-rs/commit/b5a61821d62f4161f5480e542601668ece0e26c2 which has your commit in its history. I might be missing something though.

Reply:
Added feature flag in [04188b4](https://github.com/vectordotdev/vector/pull/15375/commits/04188b4a6f2fca19caaea88c2d67b2b50b541e8f)

Reply:
Oh huh. Tracking the releases of this project is HARD :laughing: 

I guess if CI passes after adding that flag, that proves the changes are there.

On that note though... it looks like the musl cross build failed :sob: